ui/page/wallets/restore_page.go

```diff
@@ -63,10 +64,10 @@ type Restore struct {
 	keyEvent    chan *key.Event
 }
 
-// Loading lays out the loading widget with a faded background
-func NewRestorePage(l *load.Load) *Restore {
+func NewRestorePage(l *load.Load, onRestoreComplete func()) *Restore {
```

Comment:
This allows callers to determine which page should be loaded next, rather than using a channel. Need to retest to confirm that the issue that previously necessitated using a channel isn't resurrected.